### PR TITLE
add drone

### PR DESCRIPTION
**Pull request by nick-otter**

### CODE REVIEW

File: packer.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  "provisioners": [
+    {
+      "type": "ansible",
+      "playbook_file": "./playbook.yml",
+      "ansible_env_vars": [
+        "ANSIBLE_HOST_KEY_CHECKING=False",
+        "ANSIBLE_SSH_ARGS='-o ControlMaster=auto -o ControlPersist=60s -o ControlPath=/dev/shm/cp%%h-%%p-%%r'",
+        "ANSIBLE_NOCOLOR=True"
+      ],
+      "extra_arguments": [
```
Comment (chrisns) [inline packer.json]: whats the powershell for? this is a linux machine?

**Review by chrisns — CHANGES_REQUESTED** [review]

File: packer.json (hunk 1 above)
Comment (nick-otter) [reply packer.json]: updated

Comment (chrisns) [reply packer.json]: see https://github.com/UKHomeOffice/dq-packer-peering-haproxy/blob/master/packer.json#L31 you don't need all the song+dance

**Review by chrisns — CHANGES_REQUESTED** [review]

File: packer.json [hunk 2]
```diff
@@ -51,18 +51,8 @@
...
-        "ANSIBLE_SSH_ARGS='-o ControlMaster=auto -o ControlPersist=60s -o ControlPath=/dev/shm/cp%%h-%%p-%%r'",
-        "ANSIBLE_NOCOLOR=True"
-      ],
-      "extra_arguments": [
-        "--connection",
-        "packer",
-        "--extra-vars",
-        "ansible_shell_type=sh ansible_shell_executable=None user_access_key={{user `access_key`}} secret_access_key={{user `secret_key`}}"
-      ]
+      "sftp_command": "/usr/libexec/openssh/sftp-server",
```
Comment (chrisns) [inline packer.json]: does it need an sftp command?

**Review by chrisns — APPROVED** [review]

**Pull request merged**